app/client/components/cancel/contributions/contributionsCancellationFlowFinancialSaveAttempt.tsx

```diff
@@ -0,0 +1,118 @@
+import { css } from "@emotion/core";
+import { Button, LinkButton } from "@guardian/src-button";
+import { space } from "@guardian/src-foundations";
+import { SvgArrowLeftStraight } from "@guardian/src-icons";
+import { navigate } from "@reach/router";
+import * as Sentry from "@sentry/browser";
+import React, { useState } from "react";
+import {
+  isPaidSubscriptionPlan,
+  MembersDataApiItemContext
+} from "../../../../shared/productResponse";
+import { getMainPlan, isProduct } from "../../../../shared/productResponse";
+import { PRODUCT_TYPES } from "../../../../shared/productTypes";
+import { ContributionUpdateAmountForm } from "../../accountoverview/contributionUpdateAmountForm";
+import { GenericErrorMessage } from "../../identity/GenericErrorMessage";
+
+const container = css`
+  & > * + * {
+    margin-top: ${space[6]}px;
+`;
+
+const ContributionsCancellationFlowFinancialSaveAttempt: React.FC = () => {
+  const [showAmountUpdateForm, setShowUpdateForm] = useState(false);
+
+  const onUpdateConfirmed = (updatedAmount: number) => {
+    navigate(`mma_financial_circumstances/saved`, { state: { updatedAmount } });
+  };
+
+  const onReduceClicked = () => {
+    setShowUpdateForm(true);
+  };
+
+  const onCancelClicked = () => {
+    navigate(`mma_financial_circumstances/confirmed`);
+  };
```

Comment:
Not a biggie at all, and not something to hold the review up for. These functions don't need block scope.

e.g:
```
const onReduceClicked = () => {
  setShowUpdateForm(true);
}
```

could become:

```
const onReduceClicked = () => setShowUpdateForm(true);
```

Reply:
Yeah I'll change those!